packages/react-menu/config/normalize-import.spec.js

```diff
@@ -0,0 +1,91 @@
+// @ts-check
```

Comment:
nit: the other tests in the package use `.test` as an extension, might be good to be consistent

Reply:
right.

I created RFC to start using proper naming suffix - `.spec`, as we are using behaviour driven approach. Once we agree upon consistency we can re-name in batch and add proper automation checks.